lib/chess/cli/prompt.rb

```diff
@@ -41,23 +45,23 @@ def clear
       def run_command
         case @input_string
         when COMMANDS[:show_record]
+          @parameters = nil
           @case = SHOW_RECORD_COMMAD
         when COMMANDS[:exit]
+          @parameters = nil
           @case = EXIT_COMMAD
         end
       end
 
       def run_move
         case @input_string
         when PIECE_MOVE_REGEX
-          splitted_input_string = @input_string.split('')
-          @parameters = [splitted_input_string[0], splitted_input_string[1..2].join, splitted_input_string[3..].join]
-          @case = CASE_MOVE
+          build_move_response
         when QUEEN_SIDE_CASTLING_CODE
-          @parameters = QUEEN_SIDE
+          @parameters = INTENTION_IS_QUEEN_CASTLING
           @case = CASE_CASTLE
         when KING_SIDE_CASTLING_CODE
-          @parameters = KING_SIDE
+          @parameters = INTENTION_IS_KING_CASTLING
```

Comment:
I thought this was according to that change, but I didn't find it. Can you repeat me??

Reply:
constants shouldn't be verbs or sentences, but nouns. It's better to name these constants `QUEEN_CASTLING_INTENTION` instead of `INTENTION_IS_KING_CASTLING`

Reply:
ok, so what do you think about these:

```ruby
EMPTY_ORIGIN_CELL_ERROR
CELL_OCCUPED_BY_ENEMY_ERROR
CANT_CASTLING_ERROR
WRONG_INPUT_ERROR
MOVE_INTENTION
KING_CASTLING_INTENTION
QUEEN_CASTLING_INTENTION
```

Reply:
Sounds good